### PR TITLE
Fix section issues

### PR DESCRIPTION
**Pull request by saundefined**
Some more fixed issues. Not sure about the following return values sections:

`snmp_set_valueretrieval` based on [source](https://github.com/php/php-src/blob/master/ext/snmp/snmp.c#L1450)

`snmp_read_mib` based on [source](https://github.com/php/php-src/blob/master/ext/snmp/snmp.c#L1480)

`snmp_set_enum_print` based on [source](https://github.com/php/php-src/blob/master/ext/snmp/snmp.c#L1480)

### CODE REVIEW

**Comment by cmb69** [comment]
I'm not sure about these 3 functions either; the rest looks good to me.

File: reference/snmp/functions/snmp-set-valueretrieval.xml [hunk 1]
```diff
@@ -55,6 +55,12 @@
   </para>
  </refsect1>
  
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns &true;.
+  </para>
+ </refsect1>
```
Comment (Girgias) [inline reference/snmp/functions/snmp-set-valueretrieval.xml:63]: This used to return false with an E_WARNING prior to PHP 8.0.0

File: reference/snmp/functions/snmp-set-enum-print.xml [hunk 2]
```diff
@@ -32,6 +32,13 @@
    </varlistentry>
   </variablelist>
  </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns &true;.
```
Comment (Girgias) [inline reference/snmp/functions/snmp-set-enum-print.xml:39]: Ffs, this used to return false on a type error (https://heap.space/xref/PHP-7.4/ext/snmp/snmp.c?r=9690ded2#1572), so leave it as such.

**Review by Girgias — APPROVED** [review]
Looks good other then the nit on `snmp_set_valueretrieval()`

**Pull request merged**